cmake/macros/macro_deal_ii_pickup_tests.cmake

```diff
@@ -298,11 +294,15 @@ Comparison operator \"=\" expected for boolean match.\n"
               "${DEAL_II_${_feature}_VERSION}" VERSION_LESS "${_version}" ) OR
             ( "${_operator}" STREQUAL ".leq." AND
               "${DEAL_II_${_feature}_VERSION}" VERSION_GREATER "${_version}" ) )
-          CONTINUE()   # next test
+          SET(_skip_test TRUE)
         ENDIF()
       ENDIF()
     ENDFOREACH()
```

Comment:
@bangerth You missed this sneaky `ENDFOREACH()` here...

Reply:
Ah, and there I have been railing for many years that `break` and `continue` in C/C++ are dangerous because it is so easy to get things wrong when there are nested levels of loops -- and because one can so easily change the meaning of a `break/continue` when refactoring and introducing or reducing the number of nested loops. Now I'm stepping into that very same trap.

I so wished that the authors of C back in the 1960s had only allowed `break` and `continue` with a label that identifies the loop that we're working on...